examples/Training/matlab/ROIs.m

```diff
@@ -173,6 +173,124 @@
     roi = session.getUpdateService().saveAndReturnObject(roi);
     fprintf(1, 'Created ROI %g\n', roi.getId().getValue());
 
+    % Create and apply transforms to shapes and add them to ROI
+    
+    % Apply shearing alone to a rectangle (Shear factor set to 10 in the x
+    % and y dimensions in this example)
+    % create rectangle (shape object)
+    rectangle = createRectangle(0, 0, 10, 20);
+    rectangle = setShapeCoordinates(rectangle, 0, 0, 0);
+    % x and y shear factors
+    xShear = 10;
+    yShear = 10;
+    % create transform object
+    newTform = omero.model.AffineTransformI;
+    newTform.setA00(rdouble(1));
+    newTform.setA10(rdouble(yShear));
+    newTform.setA01(rdouble(xShear));
+    newTform.setA11(rdouble(1));
+    newTform.setA02(rdouble(0));
+    newTform.setA12(rdouble(0));
+    % apply transform
+    rectangle.setTransform(newTform);
+    
+    % Apply rotation alone to an ellipse object
+    % (angle of rotation set to 10 degrees)
+    % create ellipse (shape object)
+    ellipse = createEllipse(0, 0, 10, 20);
+    setShapeCoordinates(ellipse, 0, 0, 0);
+    % set angle of rotation
+    theta = 10;
+    % create transform object
+    newTform = omero.model.AffineTransformI;
+    newTform.setA00(rdouble(cos(theta)));
+    newTform.setA10(rdouble(-sin(theta)));
+    newTform.setA01(rdouble(sin(theta)));
+    newTform.setA11(rdouble(cos(theta)));
+    newTform.setA02(rdouble(0));
+    newTform.setA12(rdouble(0));
+    % apply transform
+    ellipse.setTransform(newTform);
+    
+    % Apply translation alone to a point object (10 pixels in the x and y
+    % direction)
+    % create point (shape object)
+    point = createPoint(5, 4);
+    setShapeCoordinates(point, 0, 0, 0);
+    % translation parameters
+    xTranslate = 10;
+    yTranslate = 10;
+    % create transform object
+    newTform = omero.model.AffineTransformI;
+    newTform.setA00(rdouble(1));
+    newTform.setA10(rdouble(0));
+    newTform.setA01(rdouble(0));
+    newTform.setA11(rdouble(1));
+    newTform.setA02(rdouble(xTranslate));
+    newTform.setA12(rdouble(yTranslate));
+    % apply transform
+    point.setTransform(newTform);
+    
+    % Apply a scale change alone to a polygon object (set scale factor to 10
+    % in the x  and y direction)
+    % create polygon (shape object)
+    polygon = createPolygon([1 5 10 8], [1 5 5 10]);
+    setShapeCoordinates(point, 0, 0, 0);
```

Comment:
@bramalingam You are not setting coordinates on the new polygon here (copy & paste error 'point').